frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1779,7 +1779,7 @@ With this feature enabled, the current page is factored in, resulting in the cou
                 },
             },
             {
-                text = _("Max pct. of screen width used for text items"),
+                text = _("Max size used for text items"),
```

Comment:
As you used `Items mini-bar height` below, why not have here `Items max size` or `Text items max size` so this menu item looks not the oddest among its siblings ?

Reply:
👍